queries.sql

```diff
@@ -0,0 +1,49 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+ROLLBACK;
+
+BEGIN TRANSACTION;
+DELETE FROM animals;
+ROLLBACK;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+BEGIN TRANSACTION;
+SAVEPOINT my_sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT my_sp1;
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] You did very well on this code, but there is a little issue to note. The `BEGIN;` on line 25, should have come before the code on line 23 because **_deleting all animals born after Jan 1st, 2022_** was suppose to be inside a transaction as well.

```sql
    BEGIN;

    DELETE FROM animals WHERE  date_of_birth > '2022/01/01' ;
    SAVEPOINT SP1;

    UPDATE animals SET weight_kg = weight_kg * -1; 
    ROLLBACK TO SP1;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; 

    COMMIT;
```
The [fourth ](https://user-images.githubusercontent.com/44959066/231163118-47ad7096-d0bb-4a9b-a2b5-54cf15c053d2.png) and [fifth](https://user-images.githubusercontent.com/44959066/231163115-23b41f45-4b5d-44f3-b889-f28f7a877f3f.png) screenshots should be one screenshot since both belong to the same transaction. Kindly fix this and remember to update your screenshots